build/yamls/antrea-windows-ovs-containerd.yml

```diff
@@ -0,0 +1,331 @@
+apiVersion: v1
+data:
+  Install-WindowsCNI-Containerd.ps1: |
+    $ErrorActionPreference = "Stop";    
+    mkdir -force c:/var/log/antrea
+
+    $mountPath = $env:CONTAINER_SANDBOX_MOUNT_POINT
+    $mountPath =  ($mountPath.Replace('\', '/')).TrimEnd('/') 
+    mkdir -force C:/var/run/secrets/kubernetes.io/serviceaccount
+    cp $mountPath/var/run/secrets/kubernetes.io/serviceaccount/ca.crt C:/var/run/secrets/kubernetes.io/serviceaccount
+    cp $mountPath/var/run/secrets/kubernetes.io/serviceaccount/token C:/var/run/secrets/kubernetes.io/serviceaccount
+    mkdir -force c:/opt/cni/bin/
+    mkdir -force c:/etc/cni/net.d/
+    cp $mountPath/k/antrea/cni/* c:/opt/cni/bin/
+    cp $mountPath/etc/antrea/antrea-cni.conflist c:/etc/cni/net.d/10-antrea.conflist
+    mkdir -force c:/k/antrea/bin
+    cp $mountPath/k/antrea/bin/antctl.exe c:/k/antrea/bin/antctl.exe
+    
+  Run-AntreaAgent-Containerd.ps1: |
+    $ErrorActionPreference = "Stop"
+    $mountPath = $env:CONTAINER_SANDBOX_MOUNT_POINT
+    $mountPath =  ($mountPath.Replace('\', '/')).TrimEnd('/')
+    & "$mountPath/k/antrea/bin/antrea-agent.exe" --config=$mountPath/etc/antrea/antrea-agent.conf --logtostderr=false --log_dir=c:/var/log/antrea --alsologtostderr --log_file_max_size=100 --log_file_max_num=4 --v=0
+
+  Run-AntreaOVS-Containerd.ps1: |
```

Comment:
Thanks @XinShuYang for reporting this.
I got this issue previously also.
 It is because of OVS version mismatch. We are using OVS 3.0.5 in image but on host Node OVS kernel driver installed maybe of different version which is causing this error.
Could you please verify which version of OVS kernel driver is used in the snapshot `win-initial-userspaceovs`? It should be 3.0.5 to work properly.

Reply:
@Atish-iaf The command I executed was .\Install-OVS.ps1 -InstallUserspace $false, using the script available at https://raw.githubusercontent.com/NamanAg30/antrea/ovsscript/hack/windows/Install-OVS.ps1, OVS version is 3.0.5. Additionally, could you explain why having different kernel versions would cause the above error? It seems like there are syntax errors with the script.

Reply:
It is because I have a testbed where I have same yaml and image and its working fine. 
Also, I got the same error last time which i resolved by upgrading OVS kernel version.

Reply:
Thanks, this error is fixed now.
Now the log is:
```
ovsdb-server: Failed to read from child (The pipe has been ended.
)
ovs-vsctl: unix:C:/openvswitch/var/run/openvswitch/db.sock: database connection failed (No such file or directory)
ovs-vsctl: unix:C:/openvswitch/var/run/openvswitch/db.sock: database connection failed (No such file or directory)
ovs-vswitchd: Failed to read from child (The pipe has been ended.
)
Get-Process : Cannot find a process with the name "ovsdb-server". Verify the process name and call the cmdlet again.
At C:\C\7527c84b8b62a296423c6bdf3ceb48f9a6857e83175a71005d56e93cd83a038e\var\lib\antrea-windows\Run-AntreaOVS-Container
d.ps1:23 char:11
+   if ( !( Get-Process ovsdb-server ) ) {
+           ~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (ovsdb-server:String) [Get-Process], ProcessCommandException
    + FullyQualifiedErrorId : NoProcessFoundForGivenName,Microsoft.PowerShell.Commands.GetProcessCommand

```